### ISSUE TITLE
0.9.6: monitor folder fills whole disk

### ISSUE THREAD

**Issue by smalenfant** [influxdata/influxdb#5391]
I have a cluster of 3 nodes, seems like the `/var/lib/influxdb/data/_internal/monitor` fills up on all 3 nodes. I don't think the problem was present in 0.9.4. Is is a configuration problem?

```
[ansible@cdn1cdidb0003 ~]$ df
df: `/dev/shm': No such file or directory
Filesystem     1K-blocks     Used Available Use% Mounted on
/dev/root       20511312 19336988    109364 100% /

[root@cdn1cdidb0002 monitor]# ls -l
total 17719728
-rw-r--r-- 1 influxdb influxdb 2147483648 Jan 16 00:00 1014
-rw-r--r-- 1 influxdb influxdb 3221225472 Jan 17 00:00 2542
-rw-r--r-- 1 influxdb influxdb  268435456 Jan 12 00:00 309
-rw-r--r-- 1 influxdb influxdb  268435456 Jan 13 00:00 362
-rw-r--r-- 1 influxdb influxdb  536870912 Jan 14 00:00 413
-rw-r--r-- 1 influxdb influxdb 1073741824 Jan 15 00:00 462
-rw-r--r-- 1 influxdb influxdb 6442450944 Jan 18 00:00 5057
-rw-r--r-- 1 influxdb influxdb 7516192768 Jan 19 00:00 8575
-rw-r--r-- 1 influxdb influxdb   16777216 Jan 19 13:23 9010
```

**Comment by zstyblik** [comment]
@smalenfant, please, search the closed issues. I believe this already has been reported and fixed in newer version. Or at least the issue sounds really similar.

**Comment by smalenfant** [comment]
@zstyblik Looks like this is still open #5307. Seems like turning off monitor is the only way to fix it for now?

**Comment by zstyblik** [comment]
@smalenfant no, https://github.com/influxdata/influxdb/issues/5290 and https://github.com/influxdata/influxdb/issues/5207 and may be other related issues. I'm not looking for more issues than this, sorry.

**Comment by smalenfant** [comment]
Looks like it is fixed in 0.10-nightly. I'll have to wait for a new stable build before we can fix this in production. For now I'll try disabling the monitor. Thanks.
